v3/fleet-local/worker/mesos-slave@.service

```diff
@@ -30,6 +30,7 @@ ExecStart=/usr/bin/bash -c "source /etc/profile.d/etcdctl.sh && \
     --net=host \
     --pid=host \
     --privileged \
+    -p 5051:5051 \
```

Comment:
Not clear actually if this is required the way I ended up reaching the mesos endpoint... it will connect regardless of if I add this in or not...